janusgraph-core/src/main/java/org/janusgraph/graphdb/database/management/ManagementSystem.java

```diff
@@ -436,12 +431,7 @@ public IndexType apply(@Nullable JanusGraphVertex janusgraphVertex) {
                         assert janusgraphVertex instanceof JanusGraphSchemaVertex;
                         return ((JanusGraphSchemaVertex) janusgraphVertex).asIndexType();
                     }
-                }), new Predicate<IndexType>() {
-            @Override
-            public boolean apply(@Nullable IndexType indexType) {
-                return indexType.getElement().subsumedBy(elementType);
-            }
-        }), new Function<IndexType, JanusGraphIndex>() {
+                }), indexType -> indexType.getElement().subsumedBy(elementType)), new Function<IndexType, JanusGraphIndex>() {
```

Comment:
Can the last function argument be simplified to `RelationTypeIndexWrapper::new`?

Reply:
done